### DISCUSSION TITLE
Devcontainer node startup will automatically interrupt child process

### DISCUSSION THREAD

**Discussion by As-wangyu** [vercel/turborepo#7423] [Help] [answered]
Start the node in the development container environment. After successful startup, a message "turbo repo_lib:: process:: child: child process exited normally" will be reported in the log
![微信图片_20240220100538](https://github.com/vercel/turbo/assets/57214174/32a7f24c-0062-459e-a810-ba9e5ed8c1bf)

**Comment by anthonyshew** [comment, marked as answer]
Closing as duplicate of #7427
